modules/mobile/spec/request/appointment_request_messages_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+
+RSpec.describe 'appointment_request_messages', type: :request do
+  before do
+    allow_any_instance_of(IAMUser).to receive(:icn).and_return('1012845331V153043')
+    iam_sign_in(build(:iam_user))
+    allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
+  end
+
+  describe '#index' do
+    let(:request_id) { '8a4886886e4c8e22016e5bee49c30007' }
+
+    it 'returns 200 with message details upon success' do
+      VCR.use_cassette('vaos/messages/get_messages', match_requests_on: %i[method uri]) do
+        get "/mobile/v0/appointment_requests/#{request_id}/messages",
+            headers: iam_headers, params: { appointment_request_id: request_id }
+        expected_message_body = {
+          'data' => [
+            {
+              'id' => '8a4886886e4c8e22016e5bee49c30007',
+              'type' => 'messages',
+              'attributes' => {
+                'messageText' => 'Testing',
+                'messageDateTime' => '11/11/2019 12:26:13',
+                'appointmentRequestId' => '8a4886886e4c8e22016e5bee49c30007',
+                'date' => '2019-11-11T12:26:13.931+0000'
+              }
+            }
+          ]
+        }
+        expect(response.status).to eq(200)
```

Comment:
Could we add a JSON schema check here too? It's one way we check that the shared service hasn't changed out from under us.

Reply:
Yup, done.